src/components/SatelliteManifestPanel/SatelliteManifestPanel.tsx

```diff
@@ -105,11 +112,19 @@ const SatelliteManifestPanel: FunctionComponent<SatelliteManifestPanelProps> = (
 
   const sortedRows = () => {
     const { direction, index } = sortBy;
+    /**
+     * This adjustedIndex is necessary because Patternfly
+     * has a strange quirk where, when a table has an
+     * onCollapse attribute, its index starts at 1, which throws off
+     * the sorting without the adjustment.
```

Comment:
Is this "quirk" a bug in Patternfly or an intended way for things to be?

Reply:
Hmmm it's kind of in between.  You could argue it's a bug; however, it seems several teams have coded around this so if they fixed it it would have breaking changes.  So I am not sure it is going to get fixed any time soon. 

See also here in curiosity-frontend, for ex - https://github.com/RedHatInsights/curiosity-frontend/blob/a3a563c020a40efb2f632444b9c81643b184b701/src/components/table/table.js#L116

I'm open to changing the language of the comment, but I feel like some kind of comment is warranted because it's not intuitive at all for adding the `onCollapse` attribute to shift the index by 1.

Reply:
Interesting.  Yeah, that's an awkward thing for them to fix.  Definitely wouldn't want the change going out in a patch release.  I guess just leave this code how you have it for now.  Might still be worth seeing if they have an issue out for the index problem though; it'd probably be good for them to fix it in at least a major version release sometime.